app/pages/admin.js

```diff
@@ -46,16 +70,39 @@ export default function Admin() {
                 </div>
                 <div className="flex w-screen mx-4 mt-2">
                   <div>
-                    {links.map((link, index) => (
-                      <div key={index}>
-                        {amountOfLinks > 0 &&  (
-                        <div>
-                          <span>Name</span>
-                          <input name="name" type="text" id="name" required  onBlur={change(index)}/>
-                          <span>Link</span>
-                          <input name="link" type="text" id="link" required />
-                        </div>
-                          )}
+                    {[...links.keys()].map((key) => (
+                      <div key={key} id = {key}>
+                        {links.size > 0 && (
+                          <div>
+                            <form id = {key}>
+                              <span>Name</span>
+                              <input id = {key}
+                                type="text"
+                                placeholder={getName(key)} 
```

Comment:
unsure of why have a placeholder: if these are created links they should have no placeholder and the link property as `value` prop, right?